tests/topotests/ospf_p2mp/r1/frr-p2mp.conf

```diff
@@ -1,13 +1,10 @@
 !
-!log file ospfd.log debug
 ! debug ospf event
 ! debug ospf client
 ! debug ospf lsa
 ! debug ospf packet all
 
 hostname r1
-password zebra
-log file /tmp/r1-frr.log
 ip forwarding
```

Comment:
in several places this was removed - is it a problem to leave it here?

Reply:
no not really.  Just didn't catch it.  Can I do another PR at a later time that removes the ip forwarding from a bunch of places that is not needed?